src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -82,6 +82,13 @@ and the size of the serialized proto in bytes, separated by a space.",
               .about(
                 "Output the bytes of a serialized Directory proto addressed by fingerprint.",
               )
+              .arg(
+                Arg::with_name("output-format")
+                  .long("output-format")
+                  .takes_value(true)
+                  .default_value("binary")
+                  .possible_values(&["binary", "text"]),
```

Comment:
Take it or leave it: maybe `s/text/debug/`? Shouldn't imply that this is for automated consumption.

Reply:
This is a specific format of protobufs called "text format" :)  https://github.com/google/protobuf/blob/1a7a7fca804afa1cf67f8be5e71092898ba40334/src/google/protobuf/text_format.h#L61